eng/Versions.props

```diff
@@ -212,10 +212,10 @@
     <!-- Packages from 2.1, 3.1, and 5.0 branches used for site extension build. -->
     <MicrosoftAspNetCoreAzureAppServicesSiteExtension21Version>2.1.1</MicrosoftAspNetCoreAzureAppServicesSiteExtension21Version>
     <MicrosoftAspNetCoreAzureAppServicesSiteExtension22Version>2.2.0</MicrosoftAspNetCoreAzureAppServicesSiteExtension22Version>
-    <MicrosoftAspNetCoreAzureAppServicesSiteExtension31Version>3.1.17-servicing-21318-8</MicrosoftAspNetCoreAzureAppServicesSiteExtension31Version>
+    <MicrosoftAspNetCoreAzureAppServicesSiteExtension31Version>3.1.18-servicing-21365-4</MicrosoftAspNetCoreAzureAppServicesSiteExtension31Version>
```

Comment:
@dougbu No idea if this is correct, I just grabbed the latest package version from the `dotnet3.1-internal-transport`/`dotnet-5-internal-transport` feeds

Reply:
Do _not_ change the properties in here. We want to stick w/ 3.1.17 until 3.1.18 is public and 5.0.8 until 5.0.9 is out.